src/NumericalAlgorithms/SphericalHarmonics/StrahlkorperFunctions.hpp

```diff
@@ -348,4 +348,17 @@ template <typename Fr>
 void normal_one_form(const gsl::not_null<tnsr::i<DataVector, 3, Fr>*> one_form,
                      const tnsr::i<DataVector, 3, Fr>& dx_radius,
                      const tnsr::i<DataVector, 3, Fr>& r_hat);
+
+/// @{
+/*!
+ * The linear least squares fit of the polynomial of order Order
+ * given a std::vector of Strahlkorpers to the ylm coefficients.
+ *
+ * \param strahlkorper The Strahlkorper surface.
```

Comment:
Describe all the parameters and return value in more detail.  How are the times related to the strahlkorpers?  What is the size of the returned vector?